section4/exercises/classMethods.js

```diff
@@ -28,13 +28,46 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
+  changeProtein() {
```

Comment:
Your methods here could be called `changeProteinToChicken` or `addRiceToToppings` - they do something very specific. Explore how you can use parameters to make what is changed, or added variable.

Reply:
Updated and thank you. I found myself getting concerned about the length of method names, so this is really helpful feedback.